### PR TITLE
update the prefect secret block when the github PAT is updated

### PR DESCRIPTION
**Pull request by fatchat**
Depends on https://github.com/DalgoT4D/prefect-proxy/pull/162

### CODE REVIEW

File: ddpui/api/dbt_api.py [hunk 1]
```diff
@@ -70,6 +70,19 @@ def put_dbt_github(request, payload: OrgDbtGitHub):
     org.dbt.gitrepo_access_token_secret = payload.gitrepoAccessToken
     org.dbt.save()
 
+    if payload.gitrepoAccessToken is not None and payload.gitrepoAccessToken != "":
+        gitrepo_url = payload.gitrepoUrl.replace(
```
Comment (Ishankoradia) [inline ddpui/api/dbt_api.py:74]: we need to create the secret block if the token is passed and the block does not exist

Comment (fatchat) [reply ddpui/api/dbt_api.py:74]: good catch, i have changed the edit into an upsert please see

Comment (Ishankoradia) [reply ddpui/api/dbt_api.py:74]: we also create the secret block in our db in `OrgPrefectBlockv1`. We should do a upsert in that django table also

Comment (fatchat) [reply ddpui/api/dbt_api.py:74]: ok done

**Review by Ishankoradia — APPROVED** [review]

**Pull request merged**